101-data-share-share/azuredeploy.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "account_name": {
```

Comment:
add metadata descriptions and appropriate defaultValues for all parameters

Reply:
descriptions have been added for all params